config/locales/en.yml

```diff
@@ -218,6 +218,83 @@ en:
       itself. (These scores are only available for English Wikipedia articles.)
     articles_short: Articles
 
+  items:
+    item_not_found: "This item has not been created yet."
+    item_development: (item development)
+    item_shown: "Showing %{count} of %{total}"
+    assigned: Assigned Items
+    available: Available Items
+    no_available: No Available Items
+    next: Next
```

Comment:
It would be preferable not to duplicate messages that don't have any differences between the "Item" version and "Article" version. That's the pattern we've used for the `courses` vs. `courses_generic` message trees.